v2/googlecloud-to-googlecloud/src/main/java/com/google/cloud/teleport/v2/options/SpannerChangeStreamsToPubSubOptions.java

```diff
@@ -214,4 +214,16 @@ public interface SpannerChangeStreamsToPubSubOptions extends DataflowPipelineOpt
   RpcPriority getRpcPriority();
 
   void setRpcPriority(RpcPriority rpcPriority);
+
+  @TemplateParameter.Boolean(
+      order = 17,
+      optional = true,
+      description =
+          "Whether or not to include the spanner database and instance to read the change stream from in the output message data",
```

Comment:
I assume the database name and intance name are from user specified names and not internal names. Can you confirm? Thanks.

Reply:
Thanks for pointing out, I agree. I renamed every thing from `spannerDatabase` to `spannerDatabaseId`